src/Video.tsx

```diff
@@ -177,6 +179,9 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
             base64Certificate: selectedDrm.base64Certificate,
             useExternalGetLicense: !!selectedDrm.getLicense,
             multiDrm: selectedDrm.multiDrm,
+            localSourceEncryptionKeyScheme:
```

Comment:
Thanks @KrzysztofMoch , remember to update the doc !

Reply:
Yup! will do it